src/components/application_manager/include/application_manager/resumption/resume_ctrl.h

```diff
@@ -115,11 +115,34 @@ class ResumeCtrl {
   virtual void OnSuspend() = 0;
 
   /**
+   * @brief Processes resumption data after receiving signal "Ignition Off"
+   */
+  virtual void OnIgnitionOff() = 0;
+
+  /**
    * @brief Processes resumption data after receiving signal "Awake"
    */
   virtual void OnAwake() = 0;
 
   /**
+   * @brief Retrieves value of is_suspended_
```

Comment:
@AKalinich-Luxoft 
Would be better to have description like "Checks if SDL has already received OnExitAllApplication notification with "SUSPEND" reason"